pkg/controller/storagecluster/uninstall_reconciler_test.go

```diff
@@ -154,3 +160,89 @@ func assertTestDeleteStorageClasses(t *testing.T, reconciler ReconcileStorageClu
 		assert.Equal(t, !isStorageClassExistAfterdelete, errors.IsNotFound(err))
 	}
 }
+
+func TestDeleteNodeAffinityKeyFromNodes(t *testing.T) {
+
+	testList := []struct {
+		createUserDefinedNodeAffinityKey bool
+	}{
+		// case1: verify deleteNodeAffinityKeyFromNodes deletes default NodeAffinityKey only
+		{false},
+		// case2: verify deleteNodeAffinityKeyFromNodes does not deletes user defined NodeAffinityKey
+		{true},
+	}
```

Comment:
As in https://github.com/openshift/ocs-operator/pull/794#discussion_r505915347 , make this a bit more readable.

Reply:
Done